requirements.txt

```diff
@@ -14,3 +14,4 @@ zxcvbn==1.0
 GitPython~=2.1.9
 markdown2~=2.3.5
 git+https://github.com/g2p/rfc6266#egg=rfc6266
+python-ldap~=3.2.0
```

Comment:
I would put this in the installation documentation of the LDAP Authentication Module

Reply:
done